README.md

```diff
@@ -1,223 +1,189 @@
 
-```
-dut[number]             #dut index (1..)
-dut[string]             #dut nick name, or '*' to execute command in all duts
-cmd[string]             #command to be executed
-arguments[dictionary]   #optional argument list
-        wait = <boolean>           # whether retcode is expected before continue next command. True (default) or False
-        expected_retcode = <int>    # expected return code (default=0)
-        timeout=<int>              # timeout, if no retcode receive
-```
+To run all existing test cases from the `examples` folder:
 
-## Documentation
-HTML documentation for Icetea can be built using sphinx. The source
-for the documentation is located in [doc-source](doc-source).
-For installation of sphinx see [installation](#installation).
+`> icetea --tc all --tcdir examples`
 
-A build script for the documentation is included in build_docs.py.
-It's a simple python scripts that also generates autodoc api documentation.
-Run the script with:
+**Running an example test case with hardware**
 
-`python build_docs.py`
+In this example we assume that a compatible board has been connected
+to the computer and an application binary for said board is available.
+The test case mentioned here is available in our github repository.
 
-Currently similar documentation is available in markdown format in
-[doc](doc). This documentation will be phased out and is not maintained
-as well as the sphinx documentation.
+`> icetea --tc test_cmdline --tcdir examples --type hardware --bin <path to a binary>`
 
-## Examples
+**Using metadata filters**
 
-**Note:** Following examples uses [`dummyDut`](test/dut/dummyDut.c)
-application. It works in unix systems.
-To run following examples please compile `dummyDut` first using make:
-`> make -C test/dut`.
+To run all test cases with testtype regression in the metadata:
 
-To print available parameters:
+`> icetea --testtype regression --tcdir <test case search path>`
 
-`> icetea --help`
+The following metadata filters are available:
+* test type (--testtype)
+* test subtype (--subtype)
+* feature (--feature)
+* test case name (--tc)
+* tested component (--component)
+* test case folder (--group)
 
-To run a single test case with the process dut:
+**Running a premade suite**
 
-`> icetea --tc test_cmdline --tcdir examples --type process --bin ./test/dut/dummyDut`
+Icetea supports a suite file that describes a suite of test cases
+in json format.
 
-To run all existing test cases from the `examples` folder:
+`> icetea --suite <suite file name> --tcdir <test case search path> --suitedir <path to suite directory>`
 
-`> icetea --tc all --tcdir examples --type process --bin ./test/dut/dummyDut`
+**Enabling debug level logging**
 
-To debug dut locally with [GDB](https://www.gnu.org/software/gdb/):
+Add -v or -vv to the command. -v increases the frameworks logging level
+to debug (default is info) and the level of logging in
+certain plugins and external components to info (default is warning).
```

Comment:
Which command?
Consider having the options as a bulleted list:
* -v increases....
* -vv increases...